std/random.d

```diff
@@ -1726,24 +1726,16 @@ body
     {
         immutable T u = (rng.front - rng.min) * factor;
         rng.popFront();
-        static if (isIntegral!R)
+
+        /* If RNG variates are integral, u looks always less than 1 in view of
+         * the definition of factor. However, we must consider floating-point
+         * calculation errors. Moreover, we have to check if a floating-point
```

Comment:
What does "we must consider floating-point calculation errors" mean?
Perhaps you mean, "Although the infinitely precise result is strictly less than 1.0, it will be rounded up to 1.0 if is greater than 1 - T.epsilon/2".